app/build.gradle

```diff
@@ -71,6 +72,10 @@ dependencies {
     implementation "com.squareup.okhttp3:okhttp:4.9.0"
     implementation "com.squareup.okhttp3:logging-interceptor:4.9.0"
 
+    //room dependencies
+    implementation "androidx.room:room-runtime:2.2.5"
```

Comment:
@TomChuck36 zauważyłem, że pojawiła się w nowa wersja Rooma. Jeśli chcesz pracować na nowszej wersji to podmień wersję na `2.4.1`, natomiast na tej też wszystko powinno działać. Zaktualizowałem już o tym informację w lekcji.